staging/src/k8s.io/apimachinery/pkg/runtime/helper.go

```diff
@@ -39,6 +39,7 @@ func (c unsafeObjectConvertor) ConvertToVersion(in Object, outVersion GroupVersi
 	return c.Scheme.UnsafeConvertToVersion(in, outVersion)
 }
 
+// TODO(jpbetz): This is the converter we pass into SMD. It does not appear to do any defaulting.
```

Comment:
Do really want to do anything about this? Should we keep it?

Reply:
Oops, I have a couple of these. Removing all of them.